setup/SetupStepCountry.qml

```diff
@@ -109,6 +113,8 @@ Item {
                     Haptic.playEffect(Haptic.Click);
                     countryListView.currentIndex = index;
                     wifi.countryCode = countries[index].Code;
+                    let command = "iw reg set " + countries[index].Code;
+                    launcher.launch(command);
                     console.debug("New WiFi country code: " + wifi.countryCode);
```

Comment:
This belongs into WifiControl, not qml. There is already a method `setCountryCode` which needs to handle all required steps, otherwise we'll run into issues. I can enhance the cpp backend.
We should avoid calling shell commands from qml whenever possible.

Reply:
Ok, I'll remove this then :)